### PR TITLE
Create Jupyter notebook image for codelabs

### LINKED ISSUE #1157: ugskdsjkds
jashdjasdks

### PR DESCRIPTION
**Pull request by richardsliu**
Fixes #1157.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/214)

### CODE REVIEW

**Comment by richardsliu** [comment]
Fixes kubeflow/kubeflow#1157.

File: codelab-image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+FROM gcr.io/kubeflow-images-public/tensorflow-1.8.0-notebook-cpu
```
Comment (jlewi) [inline codelab-image/Dockerfile]: Can you add a Makefile like:
https://gist.github.com/jlewi/5d323e483bf386b774130f063e683b33

Importantly we'd like to include the git hash of the code in the image metadata.

Comment (richardsliu) [reply codelab-image/Dockerfile]: Added.

**Pull request merged**